{{cookiecutter.project_name}}/.github/workflows/build.yaml

```
@@ -15,15 +15,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Set up Python 3.12
-        uses: actions/setup-python@v5
         with:
-          python-version: "3.12"
-          cache: "pip"
-          cache-dependency-path: "**/pyproject.toml"
-      - name: Install build dependencies
-        run: python -m pip install --upgrade pip wheel twine build
+          filter: blob:none
+          fetch-depth: 0
+      - name: Install the latest version of uv
+        uses: astral-sh/setup-uv@v5
       - name: Build package
-        run: python -m build
+        run: uv tool run hatch build
```

Maybe we should stick with `pyproject-build` here, as it’s the standard tool for the job like `twine` is, but I don’t feel strongly.

```suggestion
        run: uvx --from=build pyproject-build --installer=uv --wheel